### PR TITLE
Encryption at rest updates

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Addresses #4471 

Changes made:
- Added doc for `debug encryption-status`
- Updated the encryption doc with link to the debug command doc
- Added a known issue for higher CPU usage
- Updated the sidebar nav
- Removed the `experimental` label

To do: Figure out how to make the `rocksdb.encryption.algorithm` metric info show up on the Custom Charts doc

### CODE REVIEW

File: v19.1/encryption.md [hunk 1]
```diff
@@ -10,18 +10,9 @@ Data encryption and decryption is the process of transforming plaintext data to
 
 CockroachDB uses TLS 1.2 for inter-node and client-node [authentication](authentication.html) as well as setting up a secure communication channel. Once the secure channel is set up, all inter-node and client-node network communication is encrypted using a [shared encryption key](https://en.wikipedia.org/wiki/Transport_Layer_Security) as per the TLS 1.2 protocol. This feature is enabled by default for all secure clusters and needs no additional configuration.
 
-## Encryption at Rest (Experimental)(Enterprise)
+## Encryption at Rest (Enterprise)
```
Comment (mberhault) [inline v19.1/encryption.md]: yay! GA!

File: v19.1/encryption.md [hunk 2]
```diff
@@ -188,6 +180,8 @@ Information about keys is written to [the logs](debug-and-error-logs.html), incl
 * Active/old key information at startup.
 * New key information after data key rotation.
 
+Alternatively, you can use the [`cockroach debug encryption-status`](debug-encryption-status.html) command to view information about a store's encryption algorithm and store key.
```
Comment (mberhault) [inline v19.1/encryption.md]: this should say `encryption-active-key`

File: v19.1/debug-encryption-active-key.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach start --store=cockroach-data --enterprise-encryption=path=cockroach-data,key=aes-128.key,old-key=plain --insecure --certs-dir=certs
+~~~
+
+View the encryption algorithm and store key:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach debug encryption-active-key cockroach-data
```
Comment (mberhault) [inline v19.1/debug-encryption-active-key.md]: Do we want a sample output? If so, here's one:
```
$ cockroach debug encryption-active-key cockroach-data
AES128_CTR:be235c29239aa84a48e5e1874d76aebf7fb3c1bdc438cec2eb98de82f06a57a0
```

**Comment by Amruta-Ranade** [comment]
@mberhault Do we also want to document the `encryption-status` command? If yes, can you point me to the PR?

File: v19.1/debug-encryption-active-key.md (hunk 3 above)
Comment (Amruta-Ranade) [reply v19.1/debug-encryption-active-key.md]: Hehe.. just asked you if we want to document the command :)

**Comment by mberhault** [comment]
Seems we have the same question. I'm not convinced we need `encryption-status`, but it can be added if we want to be thorough. The PR was https://github.com/cockroachdb/cockroach/pull/28582
It's definitely a debug command in the sense that it just dumps a bit list of details: key metadata (never the key contents), and information about per-file encryption.

File: v19.1/debug-encryption-active-key.md (hunk 3 above)
Comment (Amruta-Ranade) [reply v19.1/debug-encryption-active-key.md]: Added the output.

**Comment by Amruta-Ranade** [comment]
As I understand it, the `encryption-active-key` command is the user-friendly command. `encryption status` seems more oriented towards the developers, and also displays the data keys, which I am not sure if we want the users to easily access. And we don't document all debug commands - just a select few. I agree that we probably shouldn't document it.

**Comment by mberhault** [comment]
Sounds good. Anyone curious will probably just check the list of available commands.

LGTM again. Thanks for the changes!

**Comment by jseldess** [comment]
___
*[v19.1/debug-encryption-active-key.md, line 2 at r4](https://reviewable.io/reviews/cockroachdb/docs/4577#-Laz_lU7D4214a5jV34y:-LazcOxvBJ-53MGyNMxR:bb7vl8z) ([raw file](https://github.com/cockroachdb/docs/blob/b4065aa802cc6a199c4b87b1ec3cb3c031d0b72f/v19.1/debug-encryption-active-key.md#L2)):*
<details><summary><i>Previously, lhirata wrote…</i></summary><blockquote>

This should be title case
</blockquote></details>

Ah, good catch. That's true for the other encryption debug page you added recently, too, Amruta.

**Pull request merged**